add introduction message to documentation (#10)
poe-platform/documentation: server-bots/poe-protocol-specification/requests/settings.md

* `server_bot_dependencies` (mapping of strings to integers): Declare what bots this bot will access through the [third party bot API](../../accessing-other-bots-on-poe.md). The default is an empty mapping. The keys in the mapping are handles of Poe bots or the string "any", indicating that the bot may invoke any other bot. The values are the number of calls to each bot that the server bot is expected to make. For example, setting this field to `{"chatgpt": 1}` declares that the bot will use a single call to [ChatGPT](https://poe.com/ChatGPT), and setting it to `{"any": 2}` declares that the bot may make up to two calls to any other bot. Poe may show this value to users, and will enforce that bots do not access other bots that they did not declare beforehand.
* `allow_attachments` (boolean): If true, allow users to send attachments with messages sent to the bot. The default is false.

> * `introduction_message` (string): Set a message that the bot will use to introduce itself to users at the start of a chat. By default, bots will have no introduction.

Reviewer: Add that this is Markdown and follows the same format as other messages.

Reviewer: Actually we might need support for setting the content type.

> * `introduction_message` (string): Set a message that the bot will use to introduce itself to users at the start of a chat. This message supports markdown rendering; by default, bots will have no introduction.

Reviewer: ```suggestion
* `introduction_message` (string): Set a message that the bot will use to introduce itself to users at the start of a chat. This message is always treated as Markdown. By default, bots will have no introduction.
```
